cognite_toolkit/_cdf_tk/loaders/__init__.py

```diff
@@ -82,6 +85,7 @@
     "classic",
     "data_models",
     "data_sets",
+    "hosted_extractors",
```

Comment:
This is only used internally in type hints, so we do not need to hide it if the feature flag for hosted extractors is not turned on.